Assets/Mirror/Runtime/NetworkBehaviour.cs

```diff
@@ -552,15 +594,18 @@ protected NetworkIdentity GetSyncVarNetworkIdentity(uint netId, ref NetworkIdent
         }
 
         [EditorBrowsable(EditorBrowsableState.Never)]
-        protected void SetSyncVar<T>(T value, ref T fieldValue, ulong dirtyBit)
+        protected bool SyncVarEqual<T>(T value, ref T fieldValue)
         {
             // newly initialized or changed value?
-            if (!EqualityComparer<T>.Default.Equals(value, fieldValue))
```

Comment:
why was the comparison moved out of SetSyncVar but not out of SetSyncVarGameObject?

Reply:
fixed